source/styles.tex

```diff
@@ -54,6 +54,10 @@
 
 \aliaspagestyle{chapter}{cpppage}
 
+\copypagestyle{cpppageone}{cpppage}
```

Comment:
I understand we need to nix the existing "cpppage" style for the first page, but do we have to put the contents into a header to start with?  Or can we just use \vspace{-1cm} and write the first-page-only stuff in the main body of the text? For once, that would put all the special contents in one place as opposed to the present solution, which splits stuff between styles.tex and intro.tex.

Reply:
The upside is that currently the special header is perfectly aligned with the normal headers. And the document title isn't really a "header". It's just a bit awkward that it has to be on the same page as Clause 1. I'm not sure if the alternaives would be any better.

We could certainly move the special logic into a macro, so that the actual intro.tex content doesn't contain it.

Reply:
Not an additional macro = additional indirection.

I was just asking to establish a direct relationship between the placement in the .tex source file and the output, given that we're talking about an "appears once" situation. Can we maybe redefine the page style on that very page, as opposed to elsewhere?